src/ansys/geometry/core/geometry/parameterization.py

```diff
@@ -202,6 +231,139 @@ def get_span(self) -> Real:
 
         return self.end - self.start
 
+    def get_relative_val(self, t: Real) -> Real:
+        """
+        Return an evaluation property of the interval, used in BoxUV.
+
+        Parameters
+        ----------
+        t : Real
+            The offset that the interval gets evaluated at.
+
+        Returns
+        -------
+        Real
+            The actual value according to the offset
+        """
+        return self.start + t * self.get_span()
+
+    def is_negative(self, tolerance: Real) -> bool:
+        """
+        Boolean value that indicates whether the current interval is negative.
+
+        Parameters
+        ----------
+        tolerance : Real
+            The accepted range since we could be working with doubles
+
+        Returns
+        -------
+        bool
+            True if negative False otherwise
+        """
+        return Accuracy.compare_with_tolerance(self.get_span(), 0, tolerance, tolerance)
+
+    @staticmethod
+    def unite(first: "Interval", second: "Interval") -> "Interval":
+        """
+        Return the union of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The union of the two intervals
+        """
+        if first.is_empty():
+            return second
+        if second.is_empty():
+            return first
+        return Interval(min(first.start, second.start), max(first.end, second.end))
+
+    @staticmethod
+    def intersect(first: "Interval", second: "Interval", tolerance: Real) -> "Interval":
+        """
+        Return the intersection of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The intersection of the two intervals
+        """
+        if first.is_empty() or second.is_empty():
+            return None  # supposed to be empty
+        intersection = Interval(max(first.start, second.start), min(first.end, second.send))
+        if not intersection.is_negative(tolerance):
+            return intersection
+        return None  # supposed to be empty
+
+    def contains_value(self, t: Real, accuracy: Real) -> bool:
+        """
+        Check if the current interval contains value t given the accuracy range.
+
+        Parameters
+        ----------
+        t : Real
+            The value of interest
+        accuracy: Real
+            The accepted range of error since we could be working with floats
+
+        Returns
+        -------
+        bool
+            True if the interval contains the value, false otherwise
```

Comment:
```suggestion
        Check if the current interval contains value ``t`` given the accuracy range.

        Parameters
        ----------
        t : Real
            Value of interest
        accuracy: Real
            Accepted range of error given that the interval could be in float values.

        Returns
        -------
        bool
            ``True`` if the interval contains the value, ``False`` otherwise.
```